modules/stampcollection.py

```diff
@@ -58,6 +74,38 @@ def update_vote(self, emoji: str, from_id: int, to_id: int,
         if recalculate:
             self.calculate_stamps()
 
+    def update_stamps_in_users_table(
+        self, user: DiscordUser, stamp_count: float
+    ) -> None:
+        """Update stamps count in Coda [users/team table](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Team_sur3i#_lu_Rc)"""
+        query = f'"Discord handle":"{get_user_handle(user)}"'
+        user_row = utils.get_user_row(query)
+        if user_row is None:
+            return
+        user_id = user_row["id"]
+        uri = f"https://coda.io/apis/v1/docs/{utils.DOC_ID}/tables/{utils.TEAM_GRID_ID}/rows/{user_id}"
```

Comment:
hm, production Coda tables should be probably updated only from production Discord server and not from the test server...

please discuss with @plexish whether we can use different columns (or different tables or whole different Coda docs) for testing

Reply:
A test copy of the whole system now exists: https://coda.io/d/AI-Test-Info_dah62XEPvpG/Admin-Panel_su93i?playModeWorkflowId=#_luBHo